src/components/PersonInfo/PersonInfo.tsx

```diff
@@ -0,0 +1,52 @@
+import classNames from 'classnames';
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+type Props = {
+  person: Person | null,
```

Comment:
why person can be null? When you iterate through the arr, if you have some item, it's should be type Person, not null